Quartz.Spi.MongoDbJobStore/MongoDbJobStore.cs

```diff
@@ -114,7 +115,7 @@ public MongoDbJobStore()
             DbRetryInterval = TimeSpan.FromSeconds(15);
         }
 
-        public void Initialize(ITypeLoadHelper loadHelper, ISchedulerSignaler signaler)
+        public Task Initialize(ITypeLoadHelper loadHelper, ISchedulerSignaler signaler, CancellationToken token = default(CancellationToken))
```

Comment:
I believe the cancellation token is not supposed to be persisted. I think it's just so that the CLR knows which caller cancelled an asynchronous operation.